packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -52,6 +53,8 @@ export const COMPOSITE_ONLY_ICONS = {
   LocalCameraSwitch: <CameraSwitch24Regular />,
   /* @conditional-compile-remove-from(stable) Chat_Notification_Icon*/
   ControlBarButtonBadgeIcon: <Circle20Filled />,
+  /* @conditional-compile-remove-from(stable): meeting/calling-composite */
```

Comment:
not blocking but should this be conditionally compiled since its only being used by the call with chat composite?

Reply:
Yes. This is a new field in a stable constant `COMPOSITE_ONLY_ICONS`  used only in a beta feature. So it must be conditionally added to the stable constant.